common/reviews/api/rush-lib.api.md

```diff
@@ -230,9 +231,13 @@ export type PackageManagerName = 'pnpm' | 'npm' | 'yarn';
 // @public
 export class PnpmOptionsConfiguration {
     // Warning: (ae-forgotten-export) The symbol "IPnpmOptionsJson" needs to be exported by the entry point index.d.ts
-    // 
+    // Warning: (ae-forgotten-export) The symbol "IPnpmOptionsRootPaths" needs to be exported by the entry point index.d.ts
+    //
     // @internal
-    constructor(json: IPnpmOptionsJson);
+    constructor(json: IPnpmOptionsJson, rootPaths: IPnpmOptionsRootPaths);
+    // Warning: (ae-forgotten-export) The symbol "StoreOptions" needs to be exported by the entry point index.d.ts
```

Comment:
Make sure you export this from `src/index.ts`

Reply:
Resolving this now, but I have one question:

If I don't mark this interface as `@public` or `@internal` (or the other various tags), it will complain.  AFAICT, it's only an interface that will be used internally, but if I mark it `@internal`, it complains about it not having an underscore.

Is this because the `PnpmOptionsConfiguration` class is marked `@public`?  Is it an issue for the interfaces to be `@public` as well?

Reply:
Yeah it's complaining because `PnpmOptionsConfiguration` is `@public` and `pnpmStore`'s type is `StoreOptions` and `pnpmStore` is a public member of `PnpmOptionsConfiguration`.

Just a thought on this from the perspective of the API review - can we rename the type to `PnpmStoreOptions`?

Reply:
I actually did that in this last pass; I just haven't committed it yet, because I was waiting for your feedback.

The main issue is with `IPnpmOptionsJson` and `IPnpmOptionsRootPaths`.  `IPnpmStoreOptions` is marked `@public`, so it's not complaining about that one.  I just had to export it from `index.ts`.

Reply:
I went ahead and marked the interfaces public... they're just types, after all.  Please let me know if that's an issue.  Also added some docs to describe the interface members.  Trying to iron out the these issue in the API docs.